src/main/java/frc/robot/subsystems/input.java

```diff
@@ -0,0 +1,37 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class input extends SubsystemBase {
+  //Initializes the motor controller
+  VictorSPX m_motor = new VictorSPX(Constants.inputPort);
+  /**
+   * Creates a new input.
+   */
+  public input() {
+
+  }
+  /**
+   * Spins the green ball sucking motor thing
+   */
+  public void greenSpin() {
+    m_motor.set(ControlMode.PercentOutput, Constants.inputSpeed);
+  }
+  /**
+   * Stops the green spinny thing
+   */
```

Comment:
Don't use the word "thing." Reference the subsystem. Example for the first comment would be "Start spinning the input motor with the speed set in Constants.java"